dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -0,0 +1,351 @@
+import { Island } from './Island';
+import { SendAMessage } from './SendAMessage.importable';
+
+type Props = {
+	format: ArticleFormat;
+	pageId: string;
+	webTitle: string;
+	byline?: string;
+	tags: TagType[];
+	primaryDateline: string;
+	secondaryDateline: string;
+	branding?: BrandingType;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	isCommentable: boolean;
+	ajaxUrl: string;
+	messageUs?: MessageUs;
+};
+
+const metaGridContainer = (showAvatar: boolean) => css`
+	display: grid;
+	grid-template-rows: auto;
+
+	${showAvatar
+		? css`
+				grid-template-columns: auto 1fr auto;
+				grid-template-areas:
+					'avatar byline comment-count'
+					'avatar byline comment-count'
+					'dateline dateline dateline';
+		  `
+		: css`
+				grid-template-columns: 1fr auto;
+				grid-template-areas:
+					'byline comment-count'
+					'byline comment-count'
+					'dateline dateline';
+		  `}
+`;
+
+const MetaGridAvatar = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: avatar;
+			width: 140px;
+			height: 140px;
+			margin-top: 6px;
+			margin-right: 10px;
+			margin-bottom: 12px;
+			margin-left: 0px;
+
+			${until.leftCol} {
+				grid-area: avatar;
+				width: 60px;
+				height: 60px;
+				margin-top: 3px;
+				margin-right: 10px;
+				margin-bottom: 12px;
+				margin-left: 0px;
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridByline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: byline;
+			width: auto;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridCommentCount = ({
+	children,
+	isPictureContent,
+}: {
+	children: React.ReactNode;
+	isPictureContent: boolean;
+}) => (
+	<div
+		data-print-layout="hide"
+		css={css`
+			grid-area: comment-count;
+			border-left: 1px solid ${themePalette('--article-border')};
+
+			${until.desktop} {
+				margin-top: ${isPictureContent ? '-6px' : '-2px'};
+				padding-top: ${isPictureContent ? '6px' : '2px'};
+				padding-left: 10px;
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const MetaGridDateline = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			grid-area: dateline;
+		`}
+	>
+		{children}
+	</div>
+);
+
+const metaPadding = (format: ArticleFormat) => {
+	if (
+		format.design === ArticleDesign.LiveBlog ||
+		format.design === ArticleDesign.DeadBlog
+	) {
+		return css`
+			${between.tablet.and.leftCol} {
+				order: 3;
+			}
+
+			padding-top: 2px;
+		`;
+	}
+
+	return css`
+		${between.tablet.and.leftCol} {
+			order: 3;
+		}
+
+		${until.mobileLandscape} {
+			padding-left: 10px;
+			padding-right: 10px;
+		}
+
+		${from.mobileLandscape} {
+			padding-left: 20px;
+			padding-right: 20px;
+		}
+
+		${from.phablet} {
+			padding-left: 0px;
+			padding-right: 0px;
+		}
+
+		padding-top: 2px;
+	`;
+};
+
+const stretchLines = (isPictureContent: boolean) => css`
+	display: block;
+
+	${!isPictureContent && until.mobileLandscape} {
```

Comment:
I think you need to extract these styles separatly as this will not be a valid media query (`false`).